clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1485,6 +1485,43 @@ default DescribeFeaturesResult describeFeatures() {
      */
     UpdateFeaturesResult updateFeatures(Map<String, FeatureUpdate> featureUpdates, UpdateFeaturesOptions options);
 
+    /**
+     * Unregisters a broker.
+     * <p>
+     * This operation is supported only on self-managed Kafka clusters.
```

Comment:
Let's emphasize that this means brokers which do not use Zookeeper.

> This operation is supported only on self-managed Kafka clusters (i.e. brokers which do not rely on Zookeeper).

Same thing for the other API.

Reply:
Done.